skopt/optimizer/gp.py

```diff
@@ -213,7 +211,7 @@ def gp_minimize(func, dimensions, base_estimator=None,
                                     noise=noise)
 
     return base_minimize(
-        func, dimensions, base_estimator=base_estimator,
+        func, space, base_estimator=base_estimator,
```

Comment:
Nice catch, but with the recent changes by @yngtodd this reduces from a bug-fix to a micro-optimization